src/control-plane/auth/cognito-auth.ts

```diff
@@ -113,13 +105,19 @@ export class CognitoAuth extends Construct implements IAuth {
       this,
       'createControlPlaneIdpFunction',
       {
-        entry: path.join(__dirname, '../../resources/custom-resources/'),
+        entry: path.join(__dirname, '../../../resources/functions/auth-custom-resource'),
         runtime: Runtime.PYTHON_3_12,
-        index: 'create_control_plane_idp.py',
+        index: 'index.py',
         handler: 'handler',
         timeout: Duration.seconds(60),
         role: lambdaIdpExecRole,
-        layers: [lambdaLayers.controlPlaneLambdaLayer],
+        layers: [
```

Comment:
minor thing, but if all functions are using the same powertools library, why not define this once:
```
const lambdaPowertoolsLayer = LayerVersion.fromLayerVersionArn(
  this,
  'LambdaPowerTools',
  lambdaPowerToolsLayerARN
)
```

and use that in all of the lambda functions:

```
...
layers: [lambdaPowertoolsLayer],
...
```

Reply:
Yes, you are right, that was silly.

Reply:
done